src/en/clouds.md

```diff
@@ -39,6 +39,19 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region, see [Creating a controller](./controllers-creating.html).
```

Comment:
Now it looks confusing. I would put this link under the global `set-default-region` command below and say like: "To set a non-default region during the creation of a controller see *link*..."